src/extension.ts

```diff
@@ -431,28 +432,28 @@ async function isFileIgnoredForFolder(workspaceFolderPath: string, gitCommand: s
   return Promise.resolve(sout.length > 0);
 }
 
+
 async function isIgnoredByScm(fileUri: string): Promise<boolean> {
-  const uriRelative = VSCode.workspace.asRelativePath(fileUri, false);
-  const command = `git check-ignore ${uriRelative}`;
-  let ignoredForAtLeastOneWorkspaceFolder = false;
-  const workspaceFolders = VSCode.workspace.workspaceFolders;
-  if (!isIterable(workspaceFolders)) {
-    return false;
-  }
-  for (const workspaceFolder of workspaceFolders) {
-    const fileIgnoredForFolder = await isFileIgnoredForFolder(workspaceFolder.uri.fsPath, command);
-    if (fileIgnoredForFolder) {
-      ignoredForAtLeastOneWorkspaceFolder = true;
-    }
-  }
-  return Promise.resolve(ignoredForAtLeastOneWorkspaceFolder);
+  return performIsIgnoredCheck(fileUri, isIgnored);
 }
 
-function isIterable(obj): boolean {
-  if (obj == null) {
-    return false;
+export async function performIsIgnoredCheck(fileUri: string,
+                                            scmCheck: (workspaceFolderPath: string, gitCommand: string) =>
+                                                Promise<boolean>): Promise<boolean> {
+  const parsedFileUri = VSCode.Uri.parse(fileUri);
+  const workspaceFolder = VSCode.workspace.getWorkspaceFolder(parsedFileUri);
+  if(workspaceFolder == null) {
```

Comment:
Code format issue. I don't remember exactly, but you should probably install prettier extension in VSCode (@jblievremont do you remember)